horace_core/sqw/@sqw/equal_to_tol_internal.m

```diff
@@ -0,0 +1,96 @@
+function [ok, mess] = equal_to_tol_internal(w1, w2, name_a, name_b, varargin)
```

Comment:
The first 30-40 lines of this `equal_to_tol_internal` look very similar to the version in `DnDBase`. Is there a way to remove the duplication?

Reply:
Probably, I've left if here intentionally for this first dnd implementation. Once the final form of the common dnd code is clear it is easier to assess whether moving everything up to the SQWDnDBase is helpful.

Most of the repeated code is argparsing.